packages/i18n/locales/en/translation.json

```diff
@@ -477,6 +478,7 @@
     "daodaoLogo": "DAO DAO Logo",
     "daodaoWithVersion": "DAO DAO v{{version}}",
     "daosLogo": "DAO's Logo",
+    "depositFiatDescription": "Fiat is converted into USDC before it is sent to the DAO's treasury. Balances may take some time to update once the transaction completes.",
```

Comment:
Not necessarily germane to this PR, but how can users tell the status of their tx? If it completes and the balance isn't updated, is there somewhere they can go to see its status?

Reply:
The Kado modal shows the transaction as complete, but it still has to send through the bridge and make it to Juno. So the tx status is on the Kado tx history, and says complete before the DAO treasury shows the token.